src/utils.ts

```diff
@@ -42,6 +43,43 @@ export function parseDisplayNameEmail(
       `The format of '${displayNameEmail}' is not a valid email address with display name`
     )
   }
-
   return {name, email}
 }
+
+export function validatelabelPatternRequirement(
+  labelPatternRequirement: string,
+  label: string
+): string | undefined {
+  const match = label.includes(labelPatternRequirement)
+  if (match) return label
+  else return undefined
+}
+
+export function parseBranchFromLabel(
+  branchPrefix: string,
+  label: string
+): string {
+  const versionMatchRegex = /[0-9]\d*(\.[0-9]\d*)*$/
+  const version = label.match(versionMatchRegex)
+  if (!version)
+    throw new Error(
+      'user did not specify release version or the release version is in an invalid format'
+    )
+  return `${branchPrefix}${version[0]}`
+}
+
+export function filterIrrelevantBranchLabels(
+  inputs: Inputs,
+  labels: string[],
+  branch: string
+): string[] {
+  return labels.filter((label: string) => {
+    if (
+      validatelabelPatternRequirement(inputs.labelPatternRequirement, label)
+    ) {
+      const branchWithoutPrefix = branch.replace(inputs.userBranchPrefix, '')
+      if (label.includes(branchWithoutPrefix)) return true
+      else return false
+    } else return true
```

Comment:
@joebowbeerxealth  Seems that the current set up automatically formats the TS for us. So linting it would be futile. 
Should we disable that feature?

Reply:
I'm not referring to formatting, I'm referring to the strained `if (true) return true else return false` and the else after a return.

Reply:
Ah, gotcha. I've updated that logic to exit early if possible. 

I'll look into an eslint equivalent.